backend/settings.py

```diff
@@ -1,18 +1,30 @@
 """
 Django settings for backend project.
 
-Generated by 'django-admin startproject' using Django 5.1.4.
+Generated by 'django-admin startproject' using Django 5.2.
 
 For more information on this file, see
-https://docs.djangoproject.com/en/5.1/topics/settings/
+https://docs.djangoproject.com/en/5.2/topics/settings/
 
 For the full list of settings and their values, see
-https://docs.djangoproject.com/en/5.1/ref/settings/
+https://docs.djangoproject.com/en/5.2/ref/settings/
 """
 
 from pathlib import Path
 import os
 from django.core.management.utils import get_random_secret_key
+
+# Try to import CSP constants for django-csp 4.0+, fallback if not available
+try:
+    from csp.constants import SELF, UNSAFE_INLINE, UNSAFE_EVAL
+    CSP_CONSTANTS_AVAILABLE = True
```

Comment:
Why do you feel that these imports won't be available? that we have to put them in try/except?